compiler/src/dmd/frontend.h

```diff
@@ -6840,7 +6840,7 @@ class TemplateDeclaration final : public ScopeDsymbol
     const char* kind() const override;
     const char* toChars() const override;
     const char* toCharsNoConstraints() const;
-    const char* toCharsMaybeConstraints(bool includeConstraints) const;
+    const char* toCharsFull(bool includeConstraints, bool showOneMember = true) const;
```

Comment:
Check for this in `template.h`

Reply:
toCharsMaybeConstraints isn't in template.h. I had forgotten to rename the first parameter, now done.

Reply:
Ping @ibuclaw please remove changes requested